app/code/Magento/Ui/Test/Mftf/Test/AdminLocalizedDateElementTest.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminLocalizedDateElementTest">
+        <annotations>
+            <stories value="Date element is localized"/>
+            <title value="Admin user should be able to create active admin user"/>
+            <description value="Admin date element should be localized."/>
+            <testCaseId value="GITHUB_27311"/>
```

Comment:
I need to ask @okolesnyk if we should use `magento/magento2#27311` syntax instead, as we have multiple repositories and projects and the same `GITHUB_XXX` can be realted to different projects.

Reply:
I am also curious about the group and severity.

Reply:
Don't worry about this now :-)
MFTF team have to review all of them and adjust the values there.

Reply:
@tdgroot I have response from @okolesnyk - `MC-29484`